src/order_cmd.cpp

```diff
@@ -1576,10 +1576,11 @@ CommandCost CmdCloneOrder(DoCommandFlag flags, CloneOptions action, VehicleID ve
 
 			/* Trucks can't copy all the orders from busses (and visa versa),
 			 * and neither can helicopters and aircraft. */
+			Station *st;
```

Comment:
Why is this not combined at line 1581? It doesn't seem to be used after the loop.